### PR TITLE
docs: fix href hightlight

### PR DESCRIPTION
**Pull request by SrNightmare09**
- fix href highlight
- update import command `basilcss/basil.css` -> `basilcss`

### CODE REVIEW

File: content/index.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Basil uses CSS variables for consistency in design. You can use these variables
 
 # Typography
 
-Links (<a> elements) are styled with the accent color. Hover effect changes the link color to the secondary color.
+Links (elements) are styled with the accent color. <a>Hover effect changes the link color to the secondary color</a>.
```
Comment (daniel-bergmann) [inline content/index.md:74]: could you find a way to insert <a> into (elements) in some way so it's clearer?

File: content/index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The framework provides an easy-to-use grid system and various utility classes th
 Installation is simple. If you are running a Javascript application, follow these steps:
 
 1. In your terminal run `yarn add basilcss` or `npm i basilcss`
-2. Go to your root file and add this line above your global CSS file `import 'basilcss/basil.css'`
+2. Go to your root file and add this line above your global CSS file `import 'basilcss'`
```
Comment (daniel-bergmann) [inline content/index.md:20]: i really like this

File: content/index.md (hunk 1 above)
Comment (SrNightmare09) [reply content/index.md:74]: sure! let me try.

**Comment by SrNightmare09** [comment]
I have made the changes you have asked for!

**Review by daniel-bergmann — APPROVED** [review]

**Comment by daniel-bergmann** [comment]
good job, keep it up!

**Pull request merged**